gateway-ha/src/main/java/io/trino/gateway/proxyserver/ProxyRequestHandler.java

```diff
@@ -143,6 +137,11 @@ public static QueryHistoryManager.QueryDetail getQueryDetailsFromRequest(Request
         return queryDetail;
     }
 
+    private static String getRemoteTarget(URI remoteUri)
```

Comment:
This change is unrelated to "Add support for request and response compression". Move to the 1st commit.